packages/components/src/mobile/color-settings/index.native.js

```diff
@@ -77,16 +90,21 @@ function ColorSettings( {
 	}
 
 	function onCustomScreenToggle( shouldShow ) {
-		LayoutAnimation.configureNext(
-			LayoutAnimation.create(
-				300,
-				LayoutAnimation.Types.easeInEaseOut,
-				LayoutAnimation.Properties.opacity
-			)
-		);
+		performLayoutAnimation();
 		setIsCustomScreen( shouldShow );
 	}
 
+	function onCustomGradientScreenToggle( shouldShow ) {
+		performLayoutAnimation();
+		setIsCustomGradientScreen( shouldShow );
+	}
+
+	function onCustomPress() {
+		if ( isSolidSegment ) {
+			onCustomScreenToggle( true );
+		} else onCustomGradientScreenToggle( true );
```

Comment:
Can we keep the same format we use for if / else statements?

```suggestion
		} else {
			onCustomGradientScreenToggle( true );
		}
```